pkg/messaging/direct_messaging.go

```diff
@@ -84,6 +85,17 @@ func (d *directMessaging) Invoke(ctx context.Context, targetAppID string, req *i
 	return d.invokeWithRetry(ctx, retry.DefaultLinearRetryCount, retry.DefaultLinearBackoffInterval, targetAppID, d.invokeRemote, req)
 }
 
+func (d *directMessaging) requestNamespace(targetAppID string) (string, error) {
+	items := strings.Split(targetAppID, ".")
+	if len(items) == 1 {
+		return d.namespace, nil
```

Comment:
whats default value for d.namespace?

Reply:
the current namespace the app is running in